docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -14,35 +14,48 @@ Examples of built-in Rancher extensions are Fleet, Explorer, and Harvester. Exam
 
 ## Prerequisites
 
-> You must log in as an admin in order to view and interact with the extensions management page.
+> 1. You must log in as an administrator to view and interact with the extensions management page.
+> 2. You must enable extension support.
 
-## Installing Extensions
+## Enabling Extension Support in Rancher
 
-1. Click **☰ > Extensions** under **Configuration**.
+In Rancher v2.9.0 and later, extension support is now included in Rancher Manager by default.
+
+You can confirm if extension support is enabled by checking the `uiextension` feature flag. Any changes to this feature flag cause the Rancher pod to restart.
+
+When extension support is enabled for the first time, it creates resources, such as the CRD, so that UI Extensions can work.
+When extension support is disabled, it disables the endpoints and does not cache any files. However, it does not remove any CRs or delete any extensions that were installed before.
+If re-enabled, it exposes the required endpoints again and creates CRDs as needed. The extensions that were already installed load after the Rancher pod restarts.
```

Comment:
```suggestion
When you enable extension support for the first time, it creates resources, such as the CRDs, so that UI Extensions can work. When extension support is disabled, it disables the endpoints and does not cache any files. However, it does not remove any CRs or delete any extensions that were installed before. If re-enabled, it exposes the required endpoints again and creates CRDs as needed. The extensions that were already installed load after the Rancher pod restarts.
```